roll/__init__.py

```diff
@@ -382,9 +397,12 @@ def write(self, *args):
 
 
 Route = namedtuple('Route', ['payload', 'vars'])
+Protocols = {
+    'websocket': WebsocketHandler,
+}
 
 
-class Roll:
+class Roll(dict):
```

Comment:
I used it to store the websockets upon creation. If you mean : would it be better to have an attribute rather than a container logic, maybe ? I think it's rather nice to be able to store, for an extension, something in the app itself. It's probably out of scope, I can remove it.